docs/sources/koji.rst

```diff
@@ -77,6 +78,21 @@ Push items produced in this case may include:
 - :class:`~pushsource.OperatorManifestPushItem` - if images have attached operator
   manifest archives
 
+Acessing virtual machine images
+...............................
+
+Use the ``vmi_build`` parameter to request virtual machine images from one or
+more builds. Builds can be specified by NVR. Builds should be produced by
+`OSBS`_ or should use compatible metadata.
+
+``koji:https://koji.fedoraproject.org/kojihub?vmi_build=fedora-azure-37-1.7.x86_64.vhd.xz``
```

Comment:
Does this actually exist in fedora koji?  If so, can you please give me a link, as I can't find it.  Or if it doesn't exist, can you please give me a link to a representative build in any koji?

The main reason I'm asking is because this looks like a filename, not a build NVR, while all the docs say it should be a build NVR.  I want to see a real build so I can understand what one of these builds looks like.

Reply:
I check fedora koji and was not able to find something which would look like from ImageBuilder for AWS or Azure
* https://koji.fedoraproject.org/koji/builds?start=0&order=-build_id&type=image

Reply:
I've changed it to Image Builder but IDK if there's a link to it that we can include in documentation.

Please let me know if it exists some online documentation for it, otherwise it will be just plain text.

Reply:
Is it this? https://www.osbuild.org/guides/image-builder-service/image-builder-koji.html

Also, even if there is no real example we can link to in Koji now, can we please fix the vmi_build parameter to be more accurate?

I started this thread because "fedora-azure-37-1.7.x86_64.vhd.xz" looks like the name of an archive attached to a koji build, but the code expects a build NVR or ID, not a filename.  So the example should instead look something like "fedora-azure-37-1.7", which is a feasible build NVR, shouldn't it?